### PR TITLE
Add sentence about rustup update on getting started page, (issue #699)

### PR DESCRIPTION
**Pull request by lislis**
Hello fine people!
I've browsed the issues and found #699.
Here you go!
It looks like this:

Let me know what you think or if you want different copy. I could also provide the German locale if desired. 

Edit: updated the screenshot (again)

### CODE REVIEW

File: locales/en-US/learn.ftl [hunk 1]
```diff
@@ -74,6 +74,8 @@ learn-play-button = Try Rust without installing
 
 learn-install-rustup-header = Rustup: the Rust installer and version management tool
 learn-install-rustup = The primary way that folks install Rust is through a tool called Rustup, which is a Rust installer and version management tool.
+learn-install-rustup-update-header = Rustup do date?
```
Comment (Manishearth) [inline locales/en-US/learn.ftl]: "do date?"

Comment (lislis) [reply locales/en-US/learn.ftl]: typo :D

File: locales/en-US/learn.ftl [hunk 2]
```diff
@@ -74,6 +74,8 @@ learn-play-button = Try Rust without installing
 
 learn-install-rustup-header = Rustup: the Rust installer and version management tool
 learn-install-rustup = The primary way that folks install Rust is through a tool called Rustup, which is a Rust installer and version management tool.
+learn-install-rustup-update-header = Rustup to date?
```
Comment (Manishearth) [inline locales/en-US/learn.ftl]: Should be "up to date"?

File: templates/learn/get-started.hbs [hunk 3]
```diff
@@ -20,7 +20,10 @@
     <h3>{{fluent "learn-install-rustup-header"}}</h3>
     <p>{{fluent "learn-install-rustup"}}</p>
     {{> components/tools/rustup }}
-    <br /><br />
+    <br />
+    <h3>{{fluent "learn-install-rustup-update-header"}}</h3>
+    <p>{{fluent "learn-install-rustup-update"}} <code>rustup update</code>.</p>
```
Comment (Manishearth) [inline templates/learn/get-started.hbs]: There's no guarantee that other languages will have sentence structures allowing this. This should instead be

```hbs
{{#fluent "learn-install-rustup-update"}}{{#fluentparam "update-command"}}<code>rustup update</code>{{/fluentparam}}{{/fluent}}
```

and

```ftl
learn-install-rustup-update = Rust updates very frequently. If you have installed Rustup some time ago, chances are your Rust version is out of date. Get the latest version of Rust by running { $update-command }
```

Comment (lislis) [reply templates/learn/get-started.hbs]: Ah, thanks for that comment, I didn't think about that!

**Comment by XAMPPRocky** [comment]
@Manishearth Can you give this one last review, and I'll merge it in?

File: locales/en-US/learn.ftl [hunk 4]
```diff
@@ -74,6 +74,8 @@ learn-play-button = Try Rust without installing
 
 learn-install-rustup-header = Rustup: the Rust installer and version management tool
 learn-install-rustup = The primary way that folks install Rust is through a tool called Rustup, which is a Rust installer and version management tool.
+learn-install-rustup-update-header = Rustup up to date?
```
Comment (Manishearth) [inline locales/en-US/learn.ftl]: "Is rustup up to date?" perhaps? I'm not sure, r+ either way

Comment (XAMPPRocky) [inline locales/en-US/learn.ftl]:
```suggestion
learn-install-rustup-update-header = Is rustup up to date?
```

**Comment by lislis** [comment]
Thanks for the  feedback on wording! I've updated the copy and rebased on current master

**Comment by XAMPPRocky** [comment]
Thank you for your PR, and congrats on your first contribution! 🎉

**Pull request merged**